src/types.ts

```diff
@@ -16,7 +13,7 @@ export type ThirdPartyTag = {
 	loaded?: boolean;
 	onLoad?: () => void;
 	shouldRun: boolean;
-	sourcepointId?: string;
+	name?: string;
```

Comment:
Would it be worth importing the `enum` type from `@guardian/consent-management-platform`?